components/resources/library/src/androidMain/kotlin/org/jetbrains/compose/resources/ImageResources.android.kt

```diff
@@ -0,0 +1,8 @@
+package org.jetbrains.compose.resources
+
+import android.graphics.BitmapFactory
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+
+internal actual fun ByteArray.toImageBitmap(): ImageBitmap =
+    BitmapFactory.decodeByteArray(this, 0, size).asImageBitmap()
```

Comment:
Does it have any thread limitations? Will Android warn about it in strict mode?

Reply:
No. Android works with resources on UI thread, AFAIK

Reply:
I checked it and so far all is fine
```kotlin
StrictMode.setThreadPolicy(StrictMode.ThreadPolicy.Builder()
    .detectDiskReads()
    .detectDiskWrites()
    .detectAll()
    .penaltyLog()
    .penaltyDeath()
    .build())
```